boards/xtensa/heltec_wifi_lora32_v2/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# HELTEC ESP32 board configuration
+
+# Copyright (c) 2017 Intel Corporation
```

Comment:
I think you can update this copyright to 2021. Eventually change to the company you work for. If none, ok.